docs/src/main/sphinx/release/release-418.md

```diff
@@ -0,0 +1,61 @@
+# Release 418 (17 May 2023)
+
+## General
+
+* Add support for `EXECUTE IMMEDIATE`. ({issue}`17341`)
+* Improve performance of joins on partitioned columns. ({issue}`14493`)
+* Fix failure when invoking `current_timestamp`. ({issue}`17455`)
+
+## BigQuery connector
+
+* Add support for adding labels to BigQuery jobs started by Trino as part of
+  query processing. The name and value of the label can be configured via the
+  `bigquery.job.label-name` and `bigquery.job.label-format` catalog
+  configuration properties, respectively. ({issue}`16187`)
+
+## Delta Lake connector
+
+* Add support for DML statements for tables with an `id` column mapping. ({issue}`16600`)
+* Add `table_changes` table function. ({issue}`16205`)
+
+## Hive connector
+
+* Improve performance of querying `information_schema.tables` when using the
+  Hive metastore. ({issue}`17127`)
+* Enable the optimized Parquet writer by default. This can be disabled with
+  the `parquet.optimized-writer.enabled` configuration property or
+  the `parquet_optimized_writer_enabled` session property. ({issue}`17393`)
+* Share the value set by the
+  `hive.per-transaction-file-status-cache.max-retained-size` configuration
+  property across all running queries. ({issue}`17432`)
+* Remove the `temporary_staging_directory_enabled` and
+  `temporary_staging_directory_path` session property overrides for the
+  `temporary-staging-directory-enabled` and `temporary-staging-directory-path`
+  configuration properties. ({issue}`17390`)
+* Fix query failure for S3 objects with an empty `Content-Type` header. ({issue}`16546`)
```

Comment:
"empty `Content-Type` header" is a low level detail. Is there anything more abstract that describes the conditions under which the problem occurs?

Otherwise, I would just say something like:

"Fix failure when querying text files in S3 if the native reader is enabled".